### PR TITLE
change layer collection api to return collections instead of items

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[GEO BON] 1.x LayerAPI umsetzen](https://trello.com/c/W4pWeL0N/919-geo-bon-1x-layerapi-umsetzen)

File: services/src/layers/layer.rs [hunk 1]
```diff
@@ -67,12 +66,15 @@ pub struct LayerDefinition {
...
-    id: LayerCollectionId,
-    name: String,
-    description: String,
-    items: Vec<CollectionItem>,
+    pub id: ProviderLayerCollectionId,
+    pub name: String,
+    pub description: String,
+    pub items: Vec<CollectionItem>,
+    pub entry_label: Option<String>, // a common label for the collection's entries, if there is any // TODO: separate labels for collections and layers?
+    pub properties: Vec<(String, String)>,
```
Comment (ChristianBeilschmidt) [inline services/src/layers/layer.rs:77]: Why did you move this?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/layers/layer.rs (hunk 1 above)
Comment (michaelmattig) [reply services/src/layers/layer.rs:77]: it makes much more sense this way, because they belong to the collection that is currently being listed. When I render a collection in the UI, I want to print the properties and I want to display the entryLabel. Before the change I had to take the information from the parent collection's corresponding child entry.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge
